plugins/rikki/heroeslounge/classes/heroes/HeroUpdater.php

```diff
@@ -130,13 +130,7 @@ public static function getHeroesList()
 
     public static function getHero($hero_name)
     {
-        // There are naming exceptions for TLV and Cho.
-        if ($hero_name == 'thelostvikings') {
-            $hero_name = 'lostvikings';
-        } else if ($hero_name == 'cho') {
-            $hero_name = 'chogall';
-        }
-
+        $hero_name = Self::getHeroNameForHeroesPatchNotes($hero_name);
```

Comment:
Not sure if this causes the function to have a side effect that changes $hero_name - maybe better to use $hpn_hero_name